keps/sig-network/563-dual-stack/README.md

```diff
@@ -1299,3 +1311,146 @@ This capability will move to stable when the following criteria have been met.
 * e2e test successfully running on two platforms
 * testing ingress controller infrastructure with updated dual-stack services
 * dualstack tests run as pre-submit blocking for PRs
+
+
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+
+* **How can this feature be enabled / disabled in a live cluster?**
+  - [X] Feature gate (also fill in values in `kep.yaml`)
+    - Feature gate name: IPv6DualStack
+    - Components depending on the feature gate: kube-apiserver, kube-controller-manager, kube-proxy, and kubelet
+  - [ ] Other
+    - Describe the mechanism:
+    - Will enabling / disabling the feature require downtime of the control
+      plane?
+    - Will enabling / disabling the feature require downtime or reprovisioning
+      of a node? (Do not assume `Dynamic Kubelet Config` feature is enabled).
+
+* **Does enabling the feature change any default behavior?**
+  No. Pods and Services will remain single stack until cli flags have been modified
+  as described in this KEP. Once modified, existing and new services will remain
+  single stack until user requests otherwise. Pods will become dual-stack however
```

Comment:
Pods will only "become dual stack" if their net driver is actually allocating both families, which might be happening already and just not exposed or might only start happening once the gate is opened and dual-stack CIDRs are allocated per node.

Reply:
Do we need to ref a hack used by super users on specific CNIs?

Reply:
We "forgot" to feature gate podips, so they are always present. They really are out of kubernetes and depend on the CRI/CNi , what does depend is the IP order reported in the API https://github.com/kubernetes/kubernetes/pull/97979

Reply:
I don't think it is a real risk, and I guess the question is "default behavior", so OK.